studio/components/interfaces/Settings/Addons/ComputeInstanceSidePanel.tsx

```diff
@@ -227,7 +238,7 @@ const ComputeInstanceSidePanel = () => {
                         )
                       }}
                     >
-                      <Image
+                      <img
```

Comment:
Why not `<Image>`?

Reply:
The styling was messed up and the borders were missing

Before
![image](https://github.com/supabase/supabase/assets/14073399/2ca39685-38a9-4704-b450-af3690a576d0)


After
![image](https://github.com/supabase/supabase/assets/14073399/ac74b8a8-e7ca-4903-9c5b-586016a57b5f)

Reply:
We use `img` for the other add-on such as pitr and custom-domain